playwright.config.ts

```diff
@@ -0,0 +1,60 @@
+import { defineConfig, devices } from '@playwright/test';
+import path, { dirname } from 'path';
+
+import { PluginOptions } from '@grafana/plugin-e2e';
+
+const testDirRoot = 'e2e/plugin-e2e/';
+
+export default defineConfig<PluginOptions>({
+  fullyParallel: true,
+  projects: [
+    // Login to Grafana with admin user and store the cookie on disk for use in other tests
+    {
+      name: 'authenticate',
+      testDir: `${dirname(require.resolve('@grafana/plugin-e2e'))}/auth`,
```

Comment:
Could be an idea to assign this to a variable at the top of the file, and then assign that variable to `testDIr`, wdyt?